tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+    "compilerOptions": {
+      "target": "es5",
+      "lib": ["dom", "dom.iterable", "esnext"],
+      "allowJs": true,
+      "skipLibCheck": true,
+      "esModuleInterop": true,
+      "allowSyntheticDefaultImports": true,
+      "strict": true, // Set to true to keep strict checks
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/taulant-eulerity/Canva/pull/4#issuecomment-2181107519>#49b08d</a><span> - 06/20/2024, 04:35 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The 'strict' option is set to true, but several strict type-checking options like 'noImplicitAny', 'strictNullChecks', 'strictFunctionTypes', 'strictBindCallApply', 'strictPropertyInitialization', 'noImplicitThis', and 'alwaysStrict' are set to false. This inconsistency can lead to unexpected type-checking behavior and potential runtime errors.
</div>
<div id="fix">
<b>Fix:</b> Align the strict type-checking options with the 'strict' setting by setting them to true. This ensures consistent and predictable type-checking behavior.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -9,16 +9,16 @@
       "strict": true, // Set to true to keep strict checks
 -     "noImplicitAny": false,
 -     "strictNullChecks": false,
 -     "strictFunctionTypes": false,
 -     "strictBindCallApply": false,
 -     "strictPropertyInitialization": false,
 -     "noImplicitThis": false,
 -     "alwaysStrict": false,
 +     "noImplicitAny": true,
 +     "strictNullChecks": true,
 +     "strictFunctionTypes": true,
 +     "strictBindCallApply": true,
 +     "strictPropertyInitialization": true,
 +     "noImplicitThis": true,
 +     "alwaysStrict": true,
```
</div>
</details>
</div>
<br>


</div>